andela/ah-frontend-valkyrie
# #162633452 Set up redux

### What does this PR do?

Setup Redux in the application

**Description of Tasks to be completed?**

 - After cloning the repo, `cd` into it and run `yarn install` to install dependencies.
 - Install the redux dev tools extension in your browser.

**How should this be manually tested?**

- Run yarn start to run the application.

- Check your redux dev tools to confirm that the state in your app appears in the dev tools.

**What are the relevant pivotal tracker stories?**

[#162633452](https://www.pivotaltracker.com/story/show/162633452)
